packages/@icon-magic/config-reader/src/config.ts

```diff
@@ -0,0 +1,102 @@
+import * as path from 'path';
+import * as glob from 'glob';
+import { loadConfigFile } from './config-loader';
+import { validateConfigSchema } from './config-validator';
+import { Icon } from '@icon-magic/icon-models';
+import { IconSetHashInterface } from '@icon-magic/icon-models/src/interface';
+import { isDirectory } from '@icon-magic/icon-models/dist/src/utils/files';
```

Comment:
We may want to consider creating an `@icon-magic/utils` package for things like this.

Reply:
I'm hoping we don't have to. I'm pulling these utils into this package as well. The more I think about a mono repo and it's modules, having each module be as complete and dependent-free as possible makes better sense.